Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -380,6 +379,29 @@ script:
       name: org_address_line_one
       required: true
       type: textArea
+    - defaultValue: 'True'
+      description: Network Configuration - Dns Name Settings - Clone DNS Names
+      name: clone_dns_names
+    - defaultValue: 'False'
+      description: Third Party - Exclude Sans
+      name: exclude_sans
+    - defaultValue: 'False'
+      name: change_management
+    - defaultValue: core
+      name: network_configuration_geography
+      type: textArea
+    - defaultValue: third-party
+      name: ra
+    - defaultValue: third-party
+      name: validation_type
+    - defaultValue: 'False'
+      name: enable_multi_stacked_certificates
+    - defaultValue: 'True'
+      name: network_configuration_quic_enabled
+    - defaultValue: enhanced-tls
+      name: network_configuration_secure_network
+    - defaultValue: 'True'
+      name: network_configuration_sni_only
```

Comment:
There is no description for these arguments, please add them

Reply:
It has been added. Please check.